API.md

```diff
@@ -2247,12 +2933,24 @@ GET `[base]/Patient?_count=2`
           {
             "system": "phone",
             "value": "(333) 333-3333",
-            "rank": 1
+            "rank": 1,
+            "extension": [
+              {
+                "url": "http://saraalert.org/StructureDefinition/phone-type",
+                "valueString": "Smartphone"
+              }
+            ]
```

Comment:
Instead of an extension for smartphone or landline, why not use `ContactPoint.use = "home"` or `ContactPoint.use = "mobile"`?

Reply:
While `use` is close to what we want to capture, it's not exactly the same. It has to do with the "purpose" of the contact point, while the Sara Alert value has to more do with the capabilities of the device. I think the main reason I decided not to use it is because we have to support "Plain Cell" as an option in addition to "Smartphone", and there isn't a nice way to do that with the `contact-point-use` code system, since they'd both only make sense as `mobile`.

Reply:
OK. Is there a functional difference from a user perspective if it is a plain cell or smartphone? They still get SMS messages right? Or does one open a webpage for the daily assessment and the other does not?

Reply:
Correct, they get a Weblink if it's a Smartphone, but not if it's a "Plain Cell"